flink-rpc/flink-rpc-akka/src/test/java/org/apache/flink/runtime/rpc/akka/AkkaRpcServiceTest.java

```diff
@@ -110,33 +111,11 @@ void testScheduleRunnable() throws Exception {
     void testExecuteRunnable() throws Exception {
```

Comment:
I'm wondering whether we should move this test into `ActorSystemScheduledExecutorAdapterTest` instead of having this being tested in `AkkaRpcServiceTest` :thinking:

Reply:
see above